core/src/main/java/org/apache/calcite/sql/SqlCall.java

```diff
@@ -97,7 +97,13 @@ public void unparse(
     if (leftPrec > operator.getLeftPrec()
         || (operator.getRightPrec() <= rightPrec && (rightPrec != 0))
         || writer.isAlwaysUseParentheses() && isA(SqlKind.EXPRESSION)) {
-      final SqlWriter.Frame frame = writer.startList("(", ")");
+      final SqlWriter.Frame frame;
+      if (operator instanceof SqlUnnestOperator) {
+        frame = writer.startList(SqlWriter.FrameTypeEnum.SIMPLE);
+      } else {
```

Comment:
Use `SqlKind.UNNEST` instead of `instanceOf`.

Reply:
thanks Danny. I changed to another approach now (mark UNENST as not a expression).